opencraft/settings.py

```diff
@@ -299,6 +299,11 @@
     'consumer': {'workers': 1, 'loglevel': logging.INFO},
 }
 
+# Instance Logs Server ########################################################
+
+INSTANCE_LOGS_SERVER_SSH_URL = env('INSTANCE_LOGS_SERVER_SSH_URL', default='logs.opencraft.com')
```

Comment:
@pcockwell, this is not a URL, just a hostname.  So `INSTANCE_LOGS_SERVER` or `INSTANCE_LOGS_SERVER_HOST` would suffice.

Reply:
Done in next commit